iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/ArgTreeImplTest.java

```diff
@@ -38,14 +38,14 @@ void matchingSimple() {
       .matches("ARG key1")
       .matches("   ARG key1")
       .matches("arg key1")
-      .matches("ARG key1=value1 key2=value2")
-      .matches("ARG key1 key2")
-      .matches("ARG key1 key2=value2")
-      .matches("ARG key1= key2=value2")
 
       .notMatches("ARGkey1")
       .notMatches("ARG")
       .notMatches("ARG ")
+      .notMatches("ARG key1=value1 key2=value2")
+      .notMatches("ARG key1 key2")
+      .notMatches("ARG key1 key2=value2")
+      .notMatches("ARG key1= key2=value2")
```

Comment:
ARG is not supposed to support multiple argument, Docker throw an error if you try to do so.